app/shared/taginput/taginput.component.html

```diff
@@ -0,0 +1,13 @@
+﻿<div>
+    <tag-input [ngModel]="[]"
```

Comment:
It was initially only intended to be an input component (Topic Creation etc.,). But yea I agree the fact that, we need for updating also. Anyways, it will still be one way data binding since we have different inputs for tag-input and autocomplete.